spread/runner.go

```diff
@@ -881,6 +881,10 @@ func (r *Runner) allocateServer(backend *Backend, system *System) *Client {
 Allocate:
 	for {
 		lerr := err
+
+		if system.Storage == 0 {
+			system.Storage = backend.Storage
+		}
```

Comment:
Right, that's the idea, but not here. When loading the project:

https://github.com/snapcore/spread/blob/52e4138/spread/project.go#L546

Also, as a general note every time we add something to the code we need to be conscious about what is around the change. The change above broke a line out from the block which it was related to (lerr := ...) and inserted something in between which was completely unrelated.

Reply:
@niemeyer right, thanks for taking a look to this. I already made the change.